semgrep_output_v1.atd

```diff
@@ -1405,6 +1418,9 @@ type ci_scan_results = {
    (* since semgrep 1.38.0 *)
    (* this data was originally sent to /complete, but we want to start sending it /results *)
    ?dependencies: ci_scan_dependencies option;
+
+   (* since semgrep 1.74.0 *)
+   ?batch_info: batch_info option;
```

Comment:
What is batched? Just the finding list? also the ignores? What about searched_paths? Also split in parts? What about
rule_ids?
And all the other fields, they will be present in each batch?

Seems like it would be better to use a different record for those batch things.